src/main/java/views/blackjack/blackjack.ftl.html

```diff
@@ -123,6 +123,41 @@
      });
 
 });
+
+//Function to handle hit requests from the user
+//Checks if the field of the hit button is "Hit"
+//If true then we make a post request to get a new card
+//Grab this new card, string configure it, and append
+//the new card to the users hand
+$("#dealHit").click(function(){
+    val = "";
+    data = "";
+    data = $("div.dealHit").text;
+
+    if(data == "Hit"){
+        var newCard;
+        //Send game data to get card{amount}
+        $.ajax({
+            type: "POST",
+            url: "/val/" + val,
```

Comment:
So for the url every thing after the second '/' is data to be passed along with the game. Since their is no data it would be just "/val". Also "val" is a route that we will define in Route.java that will connect to a specific method in Applicationcontroller. I feel like val is kind of vague. What does it mean? We will have to write controller/route code around it